### ISSUE TITLE
glooe helm upgrade from 1.2.2 to 1.2.10 broken

### ISSUE THREAD

**Issue by bdecoste** [solo-io/gloo#2566]
```
$ helm install glooe glooe/gloo-ee --namespace gloo-system --set-string license_key=eyJleHAiOjE2MTI0NjI5NzksImlhdCI6MTU4MDkyNjk3OSwiayI6InZuRGZEdyIsImx0IjoiZW50In0.ZSgrMp0dPNmPkRxuqPKr0Ju5B5o7tO-INqPvQalaJZs --version 1.2.2 -f value-overrides.yaml 
NAME: glooe
LAST DEPLOYED: Fri Mar  6 10:31:49 2020
NAMESPACE: gloo-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
```

```
$ helm upgrade glooe glooe/gloo-ee --namespace gloo-system --version 1.2.10
Error: UPGRADE FAILED: cannot patch "glooe-grafana" with kind PersistentVolumeClaim: PersistentVolumeClaim "glooe-grafana" is invalid: spec: Forbidden: is immutable after creation except resources.requests for bound claims
```

**Comment by marcogschmidt** [comment]
[This](https://github.com/helm/charts/issues/19870) seems to be the underlying problem.

**Comment by rickducott** [comment]
There is a workaround in the ticket, unclear if there is something we can automate / productize.

**Comment by rickducott** [comment]
Scale down both pods, leave PVCs, do the upgrade, scale the pods up again.

**Comment by kdorosh** [comment]
This was fixed in grafana 5.0.1 (https://github.com/helm/charts/pull/17806), will look into upgrade
